camel/model_backend.py

```diff
@@ -58,9 +78,16 @@ class StubModel(ModelBackend):
     r"""A dummy model used for unit tests."""
 
     def __init__(self, *args, **kwargs) -> None:
```

Comment:
Can we just not take any arguments?

Reply:
No. Because arguments are passed here `inst = model_class(model_type, model_config_dict)`